src/main/java/duke/main/Parser.java

```diff
@@ -0,0 +1,72 @@
+package duke.main;
+
+import duke.command.*;
+import duke.exception.*;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+public class Parser {
+    public static Command parse(String fullCommand) throws DukeException {
+        Command c;
+        String[] splitCommand = fullCommand.split(" ", 2);
+        String headCommand = splitCommand[0];
+        String commandDetails;
+
+        if (headCommand.equals("bye")) return new ExitCommand(null);
```

Comment:
Should conditional be put on a separate line and K&R style brackets (aka Egyptian style) be used?